database/migrations/2023_11_12_183753_meeting_data.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+
+    private string $meetingCategoryDictionaryId = 'ce12aa23-a5db-49f3-987b-d2db3ab24a3b';
+    private string $meetingTypeDictionaryId = '4cc5d6b5-5913-4e07-9fcb-09b6f3d3d2a0';
+    private string $meetingStatusDictionaryId = '3865a3c3-0038-4668-9d55-5d05b79d7fcd';
+    private string $meetingResourceDictionaryId = 'fdb0f818-aa1e-4ed9-97cc-2a3cb1b702df';
+    private string $meeting_category_attribute_id = '8111626d-130c-454d-b0c0-9fda9ab9917a';
+    private string $meeting_type_attribute_id = '5f7d5e66-03f9-4bcd-a726-fde82cf98d6f';
+    private string $meeting_status_attribute_id = 'e443e2c2-82fc-41d3-8fda-fe374e5329d3';
+    private string $meeting_resource_attribute_id = 'ff2fb31e-c992-4e2d-b543-f2768ee1c897';
+
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('meetings', function (Blueprint $table) {
+            $table->char('id', 36)->collation('ascii_bin')->primary();
+            $table->char('facility_id', 36)->collation('ascii_bin');
+            $table->char('category_dict_id', 36)->collation('ascii_bin');
+            $table->char('type_dict_id', 36)->collation('ascii_bin');
+            $table->string('name')->nullable();
+            $table->text('notes')->nullable();
+            $table->date('date');
+            $table->integer('start_minutes');
```

Comment:
Ja bym proponował konsekwentnie nazywać tę rzecz (minutę od północy ignorując zmiany czasu) "day minute". Czyli tutaj start_day_minute.

Reply:
"start_minutes" jest o tyle specyficzne, że i tak wymaga sprawdzenia w opisie jak ma uwzględniać strefy czasowe

start_day_minute może oznaczać że w tym polu jest minuta i dzień właśnie, start_days_minute lub start_minute_of_day by były czytelniejsze, ale nie wiem czy potrzebne

Reply:
Mi chodzi o to żeby traktować "day minute" jako typ. Typ reprezentujący minuty liczone w taki właśnie sposób. Czyli mam zmieną typu "day minute", chociaż technicznie rzecz biorąc jest typu number, więc ją nazywam cośtamDayMinute. Podobnie jak mam adres email, jest technicznie typu string, ale gdy go wsadzę do zmiennej to raczej nazwę ją personEmail albo email. Albo jak startTimestamp.

No, w każdym razie na frontendzie póki co nazywam taką rzecz "day minute", po prostu proponuję spójność. Mogłoby być "day's minute" pewnie równie dobrze ale wtedy będzie daysMinute i wygląda jak liczba mnoga i zamieszanie niepotrzebne. Poza tym po angielsku "day minute" no to będzie "minuta dnia", w sam raz.

I akurat ten postulat to zgłaszałem dość dawno :) W każdym razie poradzę sobie tak czy siak.

Reply:
Zrobiłem start_day_minute - wg mnie część day jest faktycznie ważna.

Reply:
no to chyba jak już ta forma, to bym jeszcze rozważył napisanie tego razem, start_dayminute

Reply:
No nie wiem, trochę dziwna pisownia a wydaje mi się, że nikt tego raczej jako day+minute nie zinterpretuje biorąc pod uwagę inne pola.

Co do stref czasowych to bym się jakoś nie przejmował, szanse na konsultację w noc zmiany czasu, między 2 a trzecią są nikłe i to nie są jakieś finansowe dane raczej, więc nieścisłości w stylu "czy to było o tej wcześniejszej 2:45 tego dnia czy o tej późniejszej") chyba są pomijalne?

Reply:
O strefach czasowych mamy przegadane wiele godzin :) Jakieś notatki z pewnego punktu gdy mieliśmy konkluzję opisane są [tu](https://docs.google.com/document/d/17HeSt-4CzFLd3FmzLdiM4MHXrhtEb3zg9bKlvYsBZNs/edit). Ogólnie wychodzi że bardzo dbamy o strefę czasową, trochę dlatego że się uparłem że niedbanie prowadzi prędzej czy później do katastrof, więc robimy porządnie. Ale oczywiście oznacza to przez większość czasu jej ignorowanie.

Moim zdaniem "start_dayminute" jest spoko.

Reply:
Ok, zmieniłem.